src/main/kotlin/org/jetbrains/research/testspark/tools/llm/generation/LLMProcessManager.kt

```diff
@@ -227,27 +227,31 @@ class LLMProcessManager(
             generatedTestsArePassing = true
 
             for (index in testCases.indices) {
-                report.testCaseList[index] = TestCase(index, testCases[index].name, testCases[index].toString(), setOf(), setOf(), setOf())
+                report.testCaseList[index] = TestCase(index, testCases[index].name, testCases[index].toString(), setOf())
             }
         }
 
-        if (processStopped(project, indicator)) return
+        if (processStopped(project, indicator)) return null
 
         // Error during the collecting
-        if (project.service<ErrorService>().isErrorOccurred()) return
+        if (project.service<ErrorService>().isErrorOccurred()) return null
 
         log.info("Result is ready")
 
-        val testSuitePresenter = TestSuitePresenter(project)
+        val testSuitePresenter = TestSuitePresenter(project, generatedTestsData)
         val testSuiteRepresentation =
             if (generatedTestSuite != null) testSuitePresenter.toString(generatedTestSuite) else null
-
+        transferToIJTestCases(report)
         saveData(
             project,
             report,
             getPackageFromTestSuiteCode(testSuiteRepresentation/*generatedTestSuite.toString()*/),
-            getImportsCodeFromTestSuiteCode(testSuiteRepresentation/*generatedTestSuite.toString()*/, project.service<ProjectContextService>().classFQN!!),
+            getImportsCodeFromTestSuiteCode(testSuiteRepresentation/*generatedTestSuite.toString()*/, projectContext.classFQN!!),
```

Comment:
Comments of type `/*[something]*/` can be removed. I forgot to remove it after refactoring.

Reply:
Addressed by [8e1a82a](https://github.com/JetBrains-Research/TestSpark/commit/8e1a82a0c15b67c955f15ec157cf57d52112d523)